modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -18240,6 +18240,7 @@ thousands-delimiter=,
 thousands-separator=Thousands Separator
 thread=Thread
 thread-as-question-by-default=Thread as Question by Default
+thread-cannot-be-deleted.-anonymimze-instead=Thread cannot be deleted. Anonyimize instead.
```

Comment:
There's a typo here, both in the key and the value (anonymimze / Anonyimize)

Reply:
Fixed